components/user-menu.tsx

```diff
@@ -42,7 +42,7 @@ export function UserMenu({ user }: UserMenuProps) {
               'use server'
               const supabase = createClient()
               await supabase.auth.signOut()
-              redirect('/login')
+              redirect('/')
```

Comment:
the middleware takes care of redirecting to the login page if the user is not authenticated. 👍